pkg/envoy/envoy.go

```diff
@@ -68,13 +68,13 @@ func mapLogLevel(level logrus.Level) string {
 	return envoyLevelMap[level]
 }
 
-type admin struct {
+type EnvoyAdminClient struct {
```

Comment:
this doesn't quite match the commit description - EnvoyAdminClient is still a struct here

Reply:
@bimmlerd 

thanks - renamed in the commit message